messages/deploy.metadata.md

```diff
@@ -78,7 +78,7 @@ Overrides your default org.
 
 # flags.metadata.summary
 
-Metadata component names to deploy. Wildcards ( * ) supported as long as you use quotes, such as 'ApexClass:MyClass*'
+Metadata component names to deploy. Wildcards ( _ ) supported as long as you use quotes, such as 'ApexClass:MyClass_'
```

Comment:
Is this correct?  Why `_` instead of the normal wildcard `*`

Reply:
No that's definitely not correct.  Not sure how that happened.  I'll put it back.  Good catch!